src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -61,33 +60,14 @@ const ChangeRequestDropdown = ({ control, name, errors }: ChangeRequestDropdownP
           control={control}
           name={name}
           render={({ field: { onChange, value } }) => (
-            <Select
+            <NERAutocomplete
               id="cr-autocomplete"
-              displayEmpty
-              renderValue={(value) => renderValues.get(Number(value))}
-              value={value}
-              onChange={(event: SelectChangeEvent<number>) => onChange(event.target.value)}
+              onChange={(_event, newValue) => onChange(newValue ? newValue.id : '')}
+              options={approvedChangeRequestOptions}
               size={'small'}
```

Comment:
use the errorMessage prop to show errors and delete the manual error message being shown below, see an example in TeamTools.tsx

Reply:
I can't use the errorMessage prop because if I do the error message shows twice on the client. But with it currently it will the error correctly if it applies

Reply:
I'm saying use the errorMessage prop and delete the FormTextHelper below that's currently displaying the error message. That'll give you one error message, with less duplicate code

Reply:
Got it